packages/expo-dev-menu/ios/DevMenuRCTBridge.h

```diff
@@ -16,7 +23,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
-@interface DevMenuRCTCxxBridgeDelegate : NSObject
+@interface DevMenuRCTAppDelegate : RCTAppDelegate
+
+- (void)createBridgeWithAdapter:(NSDictionary *_Nullable)launchOptions;
```

Comment:
nit: naming thing: is that intentional to pass `launchOptions` as `adapter`?

Reply:
Ohh, not really, I've just renamed it to `createBridgeAndSetAdapterWithLaunchOptions`